docs/docs/optimizing-plugins-with-guessjs.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Optimizing plugins with Guess.js
```

Comment:
Thanks for catching that @LekoArts

Reply:
@shannonbux Can you comment on the title? There is also an entry in the docs for: 

>- title: Optimize Prefetching with Guess.js*
    link: /docs/optimize-prefetching-with-guessjs/

Reply:
Yeah, this is a tricky one. Let's stick with "optimizing site performance with Guess.js" for now, since according to the creators, it is more than just prefetching: 
> The original goal of the article was to show how data can help in improving the user-perceived page load performance by using predictive analytics[3]. The artifacts from the article were executable node modules which can be used for predictive pre-fetching & data-driven clustering of JavaScript chunks.

> In the meantime, I talked to Addy Osmani who turned out to be exploring data-driven approach for predictive pre-fetching of web pages. After a few conversations, we saw that there’s a big intersection between what we were doing so we decided to merge everything under the hat of the project Guess.js[4]!

- https://blog.mgechev.com/2018/05/09/introducing-guess-js-data-driven-user-experiences-web/